docs/BreakingChanges/101 HtmlTextWriter does not render br element correctly.md

```diff
@@ -0,0 +1,23 @@
+## 101: HtmlTextWriter does not render <br/> element correctly
```

Comment:
{
  "Id": "101",
  "Title": "HtmlTextWriter does not render <br/> element correctly",
  "Details": "Beginning in the .NET Framework 4.6, calling `HtmlTextWriter.RenderBeginTag()` and `HtmlTextWriter.RenderEndTag()` with a `<BR />` element will correctly insert only one `BR` (instead of two)",
  "Suggestion": "If an app depended on the extra `<BR />` tag, `HtmlTextWriter.RenderBeginTag` should be called a second time. Note that this behavior change only affects apps that are targeting the .NET Framework 4.6, so another option is to target a previous version of the .NET Framework in order to get the old behavior.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/813061/htmltextwriter-does-not-render-br-element-correctly",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Web.UI.HtmlTextWriter.RenderBeginTag(System.String)",
    "M:System.Web.UI.HtmlTextWriter.RenderBeginTag(System.Web.UI.HtmlTextWriterTag)"
  ],
  "Related": null,
  "VersionBroken": "4.6",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": true,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Web.UI.HtmlTextWriter.RenderBeginTag(System.String)
- [x] M:System.Web.UI.HtmlTextWriter.RenderBeginTag(System.Web.UI.HtmlTextWriterTag)